pty/ptytest/ptytest.go

```diff
@@ -85,7 +86,7 @@ type PTY struct {
 func (p *PTY) ExpectMatch(str string) string {
 	p.t.Helper()
 
-	timeout, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	timeout, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
```

Comment:
I see some instances where the wait times have been updated. For example, if we were mapping directly, we would use ‘WaitMedium’ here?

Reply:
`WaitMedium` was a later addition, so this was a good catch!